docs/UserGuide.md

```diff
@@ -100,6 +149,17 @@ Format: `viewExpenses`
 Example of usage:
 
 `viewExpenses`
+```
++-----------------------------------------------------------------------------------------------------------------------+
+|                                                   Expense Statement                                                   |
++----------+--------------+--------------------+------------------------------+--------------------+--------------------+
+|    ID    |     Type     |        Date        |         Description          |      Category      |       Amount       |
++----------+--------------+--------------------+------------------------------+--------------------+--------------------+
+|    1     |   Expense    |     2023-09-30     |           milk tea           |     FOOD_DRINK     |       - $2.5       |
++-----------------------------------------------------------------------------------------------------------------------+
+| Total Expense: $2.5                                                                                                   |
++-----------------------------------------------------------------------------------------------------------------------+
+```
 
 ### Viewing the entire financial statement: `viewFinancialStatement`
 Shows the net cash on hand and lists each income and expense record along with its description, amount, date and (optional) category.  
```

Comment:
Can probably mention that the financial statement displayed will be sorted by date